scenarios/kubernetes_e2e.py

```diff
@@ -152,17 +175,19 @@ def main(args):
     PARSER = argparse.ArgumentParser()
     PARSER.add_argument(
         '--env-file', action="append", help='Job specific environment file')
+    PARSER.add_argument(
+        '--env-template', help='Job specific environment template')
```

Comment:
[docker compose](https://docs.docker.com/compose/environment-variables/) might be able to resolve this but it's an over-kill here. Two env file would not work, guess I'm fine with write the actual FOO.env out.